### PR TITLE
CDAP-4070 Instrumenting log appender in LogSaverPluginTest 

### PR DESCRIPTION
**Pull request by poornachandra**
JIRA - https://issues.cask.co/browse/CDAP-4070
Build - http://builds.cask.co/browse/CDAP-DUT3159-1

### CODE REVIEW

File: cdap-watchdog/src/test/java/co/cask/cdap/logging/save/LogSaverPluginTest.java [hunk 1]
```diff
@@ -380,19 +389,16 @@ private void testLogRead(LoggingContext loggingContext) throws Exception {
...
 
   private void publishLogs() throws Exception {
-    KafkaLogAppender appender = injector.getInstance(KafkaLogAppender.class);
-    new LogAppenderInitializer(appender).initialize("LogSaverTest");
-
     ByteArrayOutputStream bos = new ByteArrayOutputStream();
     StatusPrinter.setPrintStream(new PrintStream(bos));
     StatusPrinter.print((LoggerContext) LoggerFactory.getILoggerFactory());
 
-    ListeningExecutorService executor = MoreExecutors.listeningDecorator(Executors.newFixedThreadPool(3));
```
Comment (bdmogal) [inline cdap-watchdog/src/test/java/co/cask/cdap/logging/save/LogSaverPluginTest.java]: why was this changed?

**Comment by bdmogal** [comment]
minor question. Rest :+1:

File: cdap-watchdog/src/test/java/co/cask/cdap/logging/save/LogSaverPluginTest.java (hunk 1 above)
Comment (bdmogal) [inline cdap-watchdog/src/test/java/co/cask/cdap/logging/save/LogSaverPluginTest.java]: :+1:

**Comment by poornachandra** [comment]
Build is green, merging.
